qa/common/sequence_util.py

```diff
@@ -984,8 +984,19 @@ def check_setup(self, model_name):
                              _max_sequence_idle_ms * 1000)  # 5 secs
 
     def check_status(self, model_name, batch_exec, exec_cnt, infer_cnt):
-        stats = self.triton_client_.get_inference_statistics(model_name, "1")
-        self.assertEqual(len(stats.model_stats), 1, "expect 1 model stats")
+        start_time = time.time()
+        # Wait up to 10 seconds for the inference statistics to be ready
```

Comment:
Add more details in comment about movitivation, ex: time window between response return and statistics update / test failure intermittence

Reply:
Sure, done!